src/DynamicData.Tests/List/MergeChangeSetsFixture.cs

```diff
@@ -0,0 +1,451 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Concurrency;
+using System.Reactive.Disposables;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Bogus;
+using DynamicData.Kernel;
+using DynamicData.Tests.Domain;
+using DynamicData.Tests.Utilities;
+using FluentAssertions;
+using Microsoft.Reactive.Testing;
+using Xunit;
+using System.Collections.Concurrent;
+
+namespace DynamicData.Tests.List;
+
+public sealed class MergeChangeSetsFixture : IDisposable
+{
+#if DEBUG
```

Comment:
Somehow I didn't think of this solution to the whole testing time issue. Good thought.

However, I asked around for some perspective, and also had a couple things pointed out to me in VS that I wasn't aware of, so, I'm just going to rescind my whole proposal to try and keep times down. I mean, we shouldn't have 5 minute tests if we don't have a really good reason, but if it's a test that's intended to catch defects, just leave it here and set it up to run as long as it needs to, to be reliable.

Feel free to remove this, if you want.

Reply:
I've been doing this consistently because it really makes validating the basic operation of the tests easier.

I think we should do it in more places, such as the values for the Stress Tests.